### ISSUE TITLE
Invalid infores

### ISSUE THREAD

**Issue by cbizon** [RTXteam/RTX#2077]
On Translator Test, in PK "dd90caf2-f11f-44ee-8a4f-d79e28e45c4c", I found the following invalid infores as a primary knowledge source:
`infores:drugbank; infores:hmdb`

**Comment by saramsey** [comment]
Tagging @cbizon since I'm not sure how notification will work if the issue is transferred to a different project area (i.e., `RTXteam/RTX-KG2` to `RTXteam/RTX`).

**Comment by saramsey** [comment]
Hmm, testing the query graph
```
{
  "edges": {
    "t_edge": {
      "attribute_constraints": [],
      "knowledge_type": "inferred",
      "object": "on",
      "predicates": [
        "biolink:treats"
      ],
      "qualifier_constraints": [],
      "subject": "sn"
    }
  },
  "nodes": {
    "on": {
      "categories": [
        "biolink:Disease"
      ],
      "constraints": [],
      "ids": [
        "MONDO:0007827"
      ],
      "is_set": false
    },
    "sn": {
      "categories": [
        "biolink:ChemicalEntity"
      ],
      "constraints": [],
      "is_set": false
    }
  }
}
```
on `arax.ncats.io/devED`, it seems that this issue is still present there, as you can see from this response:

https://arax.test.transltr.io/api/arax/v1.4/response/5297ca5d-3ed0-4048-81f1-6c0489f07db6

**Comment by saramsey** [comment]
I can confirm that `arax.ncats.io/devED` is backed by KG2.8.3c:
<img width="890" alt="Screenshot 2023-07-30 at 2 17 32 PM" src="https://github.com/RTXteam/RTX/assets/5562107/27cb6429-eb0f-41e7-a5cf-d0dbacf9b8ba">

so perhaps this issue is in PloverDB or in Expand.

**Comment by edeutsch** [comment]
We conclude that KG2 is fine. The problem seems to be being introduced in Expand. Someone needs to go in to Expand and find the ";".join() and fix it.

Or maybe since this is an inferred edge, it may be coming in via xDTD? which lags in KG2 version.

Maybe this problem will be automatically fixed when we have an xDTD trained on KG2.8.3?
This is in process.

Is KG2.8.3/ExplainableDTD_v1.0_KG2.8.3_refresh.db good enough?
Answer: NO

Need the complete rebuild from KG2.8.3

**Comment by chunyuma** [comment]
It seems like this issue can be fixed by simply modifying a few code. We don't need to wait for the complete rebuild from KG2.8.3. I think I can fix it. I will keep update once I'm done with this.

**Comment by chunyuma** [comment]
After investigation, it seems like that this issue should have been fixed with `ExplainableDTD_v1.0_KG2.8.3_refresh.db` database. This issue is from the kg2.8.0.1 `edges_c.tsv` file. Now the kg2.8.3 tsv file doesn't have this issue. So we don't need to modify anything in the code.

**Comment by isbluis** [comment]
Cannot repro in any of our environments, so we'll consider this fixed via #2090.
Please reopen if necessary.
